dimod/include/dimod/adjmapbqm.h

```diff
@@ -19,7 +19,7 @@
 #include <utility>
 #include <vector>
 
-#include "dimod/utils.h"
+#include "utils.h"
```

Comment:
I think we should include the full path, by google style guide https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes. For dimod's c++ code the "source directory" is `dimod/include/`